### DISCUSSION TITLE
gamma index

### DISCUSSION THREAD

**Discussion by Kelvin-Ngowi** [nrc-cnrc/EGSnrc#1036] [Questions]
i am working on modelling a 15 Mv LINAC head.
i want to perfom comparison between the measurements and simulated information.
the best tool for dose comparison is gamma. index.
my question is, how do i get the access to gamma index.
is it a free software?

any help i would really appreciate

**Comment by rtownson** [comment]
I'm sure there are many tools to do this, but they are not part of EGSnrc. Try looking into CERR for matlab. It can load DICOM and 3ddose files.

**Reply by Kelvin-Ngowi** [reply]
I have tried this several times  but unfortunately it doesn't work perfectly
You may list other options if possible apart from CERR

**Reply by ojalaj** [reply]
From https://github.com/nrc-cnrc/EGSnrc/wiki/Visualizing-or-parsing-3ddose-files I'd suggest [Profeel](https://github.com/TPakar/Profeel)

**Reply by karl63** [reply]
I would reccomend slicer 3D it has as well many other useful features. If you cant' use commercial products, I guess that is the method of choice
